jax_distribution/configs/default.yaml

```diff
@@ -6,32 +6,35 @@ defaults:
   - rware_scenario: tiny-4ag-easy
 
 # --- Environment ---
-ENV_NAME: RobotWarehouse-v0
+env_name: RobotWarehouse-v0
 
-NUM_ENVS: 16  # Parallel updates / environments
-NUM_UPDATES: 10 # Number of updates
-SEED: 420
+num_envs: 16  # Number of vectorised environments per device.
+num_updates: 10 # Number of updates
+seed: 42
 
 # --- RL hyperparameters ---
-LR: 2.5e-4
-UPDATE_BATCH_SIZE: 2
-ROLLOUT_LENGTH: 16
+lr: 2.5e-4 # Leanring rate for network
+update_batch_size: 2 # Number of vectorised gradient updates per device.
+rollout_length: 16 # Number of environment steps per vectorised environment.
 
-PPO_EPOCHS: 4
-NUM_MINIBATCHES: 2
-GAMMA: 0.99
-GAE_LAMBDA: 0.95
-CLIP_EPS: 0.2
-ENT_COEF: 0.01
-VF_COEF: 0.5
-MAX_GRAD_NORM: 0.5
+ppo_epochs: 4 # Number of ppo epochs per training data batch.
+num_minibatches: 2 # Number of minibatches per ppo epoch.
+gamma: 0.99 # Discounting factor.
+gae_lambda: 0.95 # Lambda value for GAE computation.
+clip_eps: 0.2 # Clipping value for PPO updates and value function.
+ent_coef: 0.01 # Entropy regularisation term for loss function.
+vf_coef: 0.5 # Critic weight in
+max_grad_norm: 0.5 # Maximum norm of the gradients for a weight update.
 
 # --- Evaluation ---
-NUM_EVAL_EPISODES: 32
-NUM_EVALUATION: 2
-EVALUATION_GREEDY: False
-ABSOLUTE_METRIC: True
+num_eval_episodes: 32 # Episodes per evaluation
+num_evaluation: 2 # Number of evaluation intervals during training.
```

Comment:
Mostly just a question, is this `evaluation_frequency`/`evaluation_period`. Docs aren't quite clear, what do you mean by number of evaluation intervals?

Reply:
yeah maybe evaluation intervals is not clear, we either will change the name of the var to evaluation_frequency or update the comment to mention that it is related to the evaluation frequency, @RuanJohn what do you think ?

Reply:
`num_eval_episodes` is the number of evaluation episodes we want to do each time we evaluate and `num_evaluations` is the amount of evenly spaced evaluation to do during training.